assets/javascripts/discourse/templates/components/leave-community-button.hbs

```diff
@@ -0,0 +1,9 @@
+{{d-button
```

Comment:
Is it possible to combine this two components (`join-community-button` & `leave-community-button`) into one? They are almost identical.
Then the contents of the `set-category-notification-level` component can be transferred there

Reply:
I don't know 😔

Reply:
I will try to implement it, if you don't mind